src/utils/rpcQueries/viewFunction.ts

```diff
@@ -0,0 +1,51 @@
+import { providers } from 'near-api-js';
+import { printTxOutcomeLogs } from '@near-js/utils';
+
+// types
+import type { IOptions } from './types';
+
+type ViewFunctionResult =
+  | string
+  | number
+  | boolean
+  | null
+  | object
+  | unknown[]
+  | undefined;
+
+function parseJsonFromRawResponse(response: Uint8Array): ViewFunctionResult {
+  return JSON.parse(Buffer.from(response).toString());
```

Comment:
`Buffer` is not a native object in browsers. You need to use either:
* a polyfill - https://github.com/feross/buffer
* or [`TextDecoder`](https://developer.mozilla.org/en-US/docs/Web/API/TextDecoder) as it is in both the browser and Node.js (https://nodejs.org/api/util.html#class-utiltextdecoder)
* or use a library like: https://www.npmjs.com/package/@stablelib/utf8